chicago/feeds.py

```diff
@@ -1,14 +1,278 @@
-from councilmatic_core.feeds import CouncilmaticFacetedSearchFeed, BillDetailActionFeed
-from chicago.models import ChicagoBill
+import urllib
 
+from haystack.query import SearchQuerySet
 
-class ChicagoCouncilmaticFacetedSearchFeed(CouncilmaticFacetedSearchFeed):
-    # same as CouncilmaticFacetedSearchFeed but have a better item name
-    # template which uses NYCBill's friendly_name() as opposed to Bill's
-    # friendly_name()
-    title_template = "feeds/chicago_search_item_title.html"
-    bill_model = ChicagoBill
+from django.contrib.syndication.views import Feed
+from django.utils.feedgenerator import Rss201rev2Feed
+from django.urls import reverse, reverse_lazy
+from django.conf import settings
 
+from .models import Person, Bill, Organization, Event
+from .utils import to_datetime
 
-class ChicagoBillDetailActionFeed(BillDetailActionFeed):
-    title_template = "feeds/chicago_bill_actions_item_title.html"
+
+class ChicagoCouncilmaticFacetedSearchFeed(Feed):
```

Comment:
one thing that might be nice to do is simplify out class names. 

This could just be `FacetedSearchFeed` since there's no other classes out there we need to worry about colliding with. 

This is a pervasive change, but it might be nice to go ahead and do now?